api/envoy/service/tls_session_cache/v3/tls_session_cache.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package envoy.service.tls_session_cache.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.service.tls_session_cache.v3";
+option java_outer_classname = "TlsSessionCacheProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/service/tls_session_cache/v3;tls_session_cachev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: TLS session cache service]
+
+service TlsSessionCacheService {
+  rpc TlsSessionStore(TlsSessionRequest) returns (TlsSessionResponse) {
```

Comment:
The two methods should have different input and output types, and only include the fields that are applicable to that type of request